api/provider/aws/environment_update.go

```diff
@@ -110,11 +110,11 @@ func (e *EnvironmentProvider) Update(environmentID string, req models.UpdateEnvi
 	return nil
 }
 
-func (e *EnvironmentProvider) updateASGSize(autoScalingGroupName string, minSize, maxSize int64) error {
+func (e *EnvironmentProvider) updateASGSize(autoScalingGroupName string, size int) error {
 	input := &autoscaling.UpdateAutoScalingGroupInput{}
 	input.SetAutoScalingGroupName(autoScalingGroupName)
-	input.SetMinSize(minSize)
-	input.SetMaxSize(maxSize)
+	input.SetMinSize(int64(size))
```

Comment:
Don't we need to set desired as well?

Reply:
We weren't doing so before. I can confirm the scaling behavior works as expected where desired is set to min without explicitly setting it.